app/jobs/generate_payout_report_job.rb

```diff
@@ -19,6 +19,28 @@ def perform(should_send_notifications: true, final: true)
       publisher_has_unsettled_balance = false
       wallet = publisher.wallet
 
+      probi = wallet.channel_balances[publisher.owner_identifier].probi_before_fees # probi = balance
+      if probi > 0
```

Comment:
`probi.positive?` is more ruby-esque but it's not that big of deal if we leave it how it is 😄

Reply:
good to know! this is my first ruby PR so if there's other ruby conventions I should know I'm all ears :)

Reply:
Sure thing! Only thing else I see is `!wallet.address.blank?` could alternatively be written as `wallet.address.present?` that way there's not negative logic mixed into the if statement